WordPressAuthenticator/Signin/AppleAuthenticator.swift

```diff
@@ -267,3 +213,80 @@ extension AppleAuthenticator {
         ASAuthorizationAppleIDProvider().getCredentialState(forUserID: userID, completion: completion)
     }
 }
+
+// This needs to be internal, at this point in time, to allow testing.
+//
+// Notice that none of this code was previously tested. A small encapsulation breach like this is
+// worth the testability we gain from it.
+extension AppleAuthenticator {
+
+    func showSignupEpilogue(for credentials: AuthenticatorCredentials) {
+        guard let navigationController = showFromViewController?.navigationController else {
+            fatalError()
+        }
+
+        let service = loginFields.meta.appleUser.map {
+            SocialService.apple(user: $0)
+        }
```

Comment:
1/2 only changes: use `map` instead of `flatMap`. There's no need to `flatMap` because the transformation doesn't return `Optional`.